mmv1/templates/terraform/examples/access_context_manager_access_policy_scoped.tf.erb

```diff
@@ -0,0 +1,5 @@
+resource "google_access_context_manager_access_policy" "<%= ctx[:primary_resource_id] %>" {
+  parent = "organizations/123456789"
```

Comment:
We'll need to pull the org of the test run from the environment rather than use a dummy value (and a dummy value will get injected into the docs). To do that, look for `test_env_vars` for an example- there are some semi-magic keys that will do so.

Reply:
Done